ladybug/windrose.py

```diff
@@ -271,41 +283,41 @@ def north(self):
         """
         return self._north
 
-    @north.setter
+    @ north.setter
     def north(self, north):
-        assert isinstance(north, (float, int)), 'north must be a number representing ' \
-            'degrees. Got {}.'.format(north)
+        assert isinstance(north, (float, int)), 'north must be a number representing '
+        'degrees. Got {}.'.format(north)
```

Comment:
And here. You should probably do a full check through your changes to see all of the places where you might ahve done this.

Reply:
@chriswmackey thanks! Can confirm I went through and corrected all instances of this error.